### PR TITLE
Spring Data sample: small README update

### PR DESCRIPTION
**Pull request by komamitsu**
This is a followup PR for https://github.com/scalar-labs/scalardb-samples/pull/19. I think there is room to improve README and this PR handles it.

### CODE REVIEW

File: spring-data-sample/README.md [hunk 1]
```diff
@@ -129,7 +129,7 @@ Please note that starting the containers may take more than one minute.
 
 ### Initial data
 
-After the containers have loaded, you need to load the initial data by running the following command:
+After the containers have started, you need to load the initial data by running the following command:
```
Comment (komamitsu) [inline spring-data-sample/README.md:132]: We've already used term `load` for sample records in the tables on this tutorial. Also `docker` command has `load` sub-command for Docker container images. So, I think other term would avoid confusions.

File: spring-data-sample/README.md [hunk 2]
```diff
@@ -261,7 +261,7 @@ $ ./gradlew run --args="PlaceOrder 1 3:1,4:1"
 
 ## Cleanup
 
-To stop Cassandra and shut down the sample application, run the following command:
+To stop Cassandra, run the following command:
```
Comment (komamitsu) [inline spring-data-sample/README.md:264]: `docker-compose` manages only schema-loader and Cassandra not the application in this sample

Comment (josh-wong) [reply spring-data-sample/README.md:264]: Understood. LGTM!

**Review by josh-wong — APPROVED** [review]

**Review by Torch3333 — APPROVED** [review]
LGTM, thank you!

**Review by brfrn169 — APPROVED** [review]
LGTM! Thank you!

File: spring-data-sample/README.md [hunk 3]
```diff
@@ -82,8 +82,8 @@ For details, please see [Guide of Spring Data integration with ScalarDB](https:/
 All the tables are created in the `sample` namespace.
 
 - `sample.customers`: a table that manages customers' information
-- `credit_limit`: the maximum amount of money a lender will allow each customer to spend when using a credit card
-- `credit_total`: the amount of money that each customer has already spent by using the credit card
+  - `credit_limit`: the maximum amount of money a lender will allow each customer to spend when using a credit card
```
Comment (komamitsu) [inline spring-data-sample/README.md:85]: These elements should be indented

**Pull request merged**